auditbeat/tests/system/test_file_integrity.py

```diff
@@ -69,130 +90,146 @@ def test_non_recursive(self):
+                self.create_file(file3, "not reported.")
 
-            subdir = os.path.join(dirs[0], "subdir")
-            os.mkdir(subdir)
-            file3 = os.path.join(subdir, "other_file.txt")
-            self.create_file(file3, "not reported.")
+                # log entries are JSON formatted, this value shows up as an escaped json string.
+                self.wait_log_contains("\\\"deleted\\\"")
+                self.wait_log_contains("\"path\":\"{0}\"".format(escape_path(subdir)), ignore_case=True)
+                self.wait_output(3)
+                self.wait_until(lambda: any(
+                    'file.path' in obj and obj['file.path'].lower() == subdir.lower() for obj in self.read_output()))
 
-            # log entries are JSON formatted, this value shows up as an escaped json string.
-            self.wait_log_contains("\\\"deleted\\\"")
-            self.wait_log_contains("\"path\":\"{0}\"".format(escape_path(subdir)), ignore_case=True)
-            self.wait_output(3)
-            self.wait_until(lambda: any(
-                'file.path' in obj and obj['file.path'].lower() == subdir.lower() for obj in self.read_output()))
+                proc.check_kill_and_wait()
+                self.assert_no_logged_warnings()
 
-            proc.check_kill_and_wait()
-            self.assert_no_logged_warnings()
+                # Ensure all Beater stages are used.
+                assert self.log_contains("Setup Beat: auditbeat")
+                assert self.log_contains("auditbeat start running")
+                assert self.log_contains("auditbeat stopped")
 
-            # Ensure all Beater stages are used.
-            assert self.log_contains("Setup Beat: auditbeat")
-            assert self.log_contains("auditbeat start running")
-            assert self.log_contains("auditbeat stopped")
+                objs = self.read_output()
 
-            objs = self.read_output()
+                has_file(objs, file1, "430ce34d020724ed75a196dfc2ad67c77772d169")
+                has_file(objs, file2, "d23be250530a24be33069572db67995f21244c51")
+                has_dir(objs, subdir)
 
-            has_file(objs, file1, "430ce34d020724ed75a196dfc2ad67c77772d169")
-            has_file(objs, file2, "d23be250530a24be33069572db67995f21244c51")
-            has_dir(objs, subdir)
+                file_events(objs, file1, ['created', 'deleted'])
+                file_events(objs, file2, ['created'])
 
-            file_events(objs, file1, ['created', 'deleted'])
-            file_events(objs, file2, ['created'])
+                # assert file inside subdir is not reported
+                assert self.log_contains(file3) is False
 
-            # assert file inside subdir is not reported
-            assert self.log_contains(file3) is False
+        cleanup(dirs)
 
-    @unittest.skipIf(os.getenv("BUILD_ID") is not None, "Skipped as flaky: https://github.com/elastic/beats/issues/7731")
```

Comment:
do we want to always execute this? I read the issue and it seems that it is resolved but...

Reply:
🤷🏻‍♂️ if it starts flaking again I'll disable it again